doc/src/as_code.rs

```diff
@@ -193,3 +215,17 @@ impl AsCode for VariableDeclaration {
         format!("{} {}", self.ty.as_code(), self.name.name)
     }
 }
+
+impl AsCode for EnumDefinition {
+    fn as_code(&self) -> String {
+        let name = &self.name.name;
+        let values = self.values.iter().map(AsCode::as_code).join(";\n\t");
+        format!("enum {name}({values})")
```

Comment:
enum types use curly braces
https://docs.soliditylang.org/en/v0.8.17/structure-of-a-contract.html?highlight=enum#enum-types